src/enzyme_ad/jax/Implementations/HLODerivatives.td

```diff
@@ -612,11 +708,19 @@ def : HLODerivative<"MulOp", (Op $x, $y),
 
 def : HLODerivative<"NegOp", (Op $x), [(Neg (DiffeRet))]>;
 
+def : HLOInactiveOp<"NotOp">;
+
+def : HLOInactiveOp<"OrOp">;
+
 def : HLOMemoryIdentityOp<"PadOp", [], [-1], (Op $op, $padval), [
   (Slice (TypeOf $op), (DiffeRet), (PadToSliceStart), (PadToSliceLimit), (PadToSliceStride)),
   (AssertingInactiveArg)
 ]>;
 
+def : HLOInactiveOp<"PartitionIdOp">;
+
+def : HLOInactiveOp<"PopcntOp">;
```

Comment:
And here (tho partition ID is okay)

Reply:
so `PartitionIdOp` is inactive...?

Reply:
Yeah exactly that cannot contain any differential data (like threadid, etc)